src/middlewares/error-middleware.ts

```diff
@@ -0,0 +1,11 @@
+import { Request, Response, NextFunction } from 'express';
+import path from 'path';
+import Exception from '../utils/exceptions/exception';
+
+export const exceptionFilter = (err: Exception | Error, req: Request, res: Response, next: NextFunction) => {
+  console.log(err);
+  res.render(path.resolve(__dirname, '..', 'views', 'error.html'), { message: err.message });
```

Comment:
Is __dirname working with ES modules?

Reply:
Yes, it's working.

Reply:
it is interesting, because I had an error below and  need to do some imports to get it working,but maybe I missed something (const __dirname = import.meta.dirname;)

If you’re writing an ES module with the mjs extension (which forces Node into ESM mode), or with {"type": "module"} set in your package.json file, or you’re writing TypeScript and running code some other way… you might encounter this error:

    ReferenceError: __dirname is not defined in ES module scope (https://blog.logrocket.com/alternatives-dirname-node-js-es-modules/)

Reply:
yep, I see you have "esModuleInterop": true in tsconfig.json, and therefore it works I guess

Reply:
Hmm. I have't encountered such an error when refactoring to TypeScript.